src/expressServer.js

```diff
@@ -185,4 +191,43 @@ global.io = io;
 // export the database connection function
 const connectDB = require('./db/connect');
 
+// ======STRIPE WEBHOOK SETUP ======
```

Comment:
Is this section (lines 194-232) for testing purposes or something else? I'm unsure about having this route here in this file rather than a controller, and about having a separate server start up here listening on  port 4242 :thinking:

Reply:
Along with line 20, and 26-28

Reply:
for sure it's useless- inicially thought it will help to run stripe webhook